### PR TITLE
Remove survey from thank you page

### PR DESCRIPTION
**Pull request by tomrf1**
## Why are you doing this?
This survey is no longer needed.
This branch can be reverted if the survey is needed again in future.

[**Trello Card**](https://trello.com)

## Changes

* Change 1
* Change 2

#### Before:
picture 11

#### After:
picture 10

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "import/no-extraneous-dependencies": "off",
     "padded-blocks": "off",
     "import/prefer-default-export": "off",
+    "no-else-return": 0,
```
Comment (johnduffell) [inline .eslintrc]: can we make that "only one return statement on any code path" if it exists instead?  I do agree there should be as few wasys as possible to do something, but I also do disagree with this way round (but maybe it's the only option in eslint)

File: assets/components/contributionsThankYou/contributionsThankYouPage.jsx [hunk 2]
```diff
@@ -52,18 +51,16 @@ export default function ContributionsThankYouPage(props: PropTypes) {
...
-  if (props.directDebit) {
-    return (
-      <div className="component-direct-debit-details__container">
-        <DirectDebitDetails {...props.directDebit} />
-        <ContributionsSurveySection />
-      </div>
-    );
-  }
-  // recurring non-DD
-  return <EmailConfirmation />;
```
Comment (johnduffell) [inline assets/components/contributionsThankYou/contributionsThankYouPage.jsx]: https://github.com/guardian/support-frontend/pull/1015/commits
I was having fun with the same file only the other week

File: assets/components/contributionsThankYou/contributionsThankYouPage.jsx [hunk 3]
```diff
@@ -51,16 +51,14 @@ export default function ContributionsThankYouPage(props: PropTypes) {
...
-      return <EmailConfirmation />;
-    }
+  // recurring
+  if (props.directDebit) {
+    return (
+      <div className="component-direct-debit-details__container">
+        <DirectDebitDetails {...props.directDebit} />
+      </div>
+    );
   }
```
Comment (tomrf1) [inline assets/components/contributionsThankYou/contributionsThankYouPage.jsx]: I would prefer an `else` block here, but the linter doesn't like it

File: assets/components/contributionsThankYou/contributionsThankYouPage.jsx [hunk 4]
```diff
@@ -52,15 +51,11 @@ export default function ContributionsThankYouPage(props: PropTypes) {
 // ----- Auxiliary Components ----- //
 
 function BodyCopy(props: PropTypes) {
```
Comment (tsop14) [inline assets/components/contributionsThankYou/contributionsThankYouPage.jsx]: could this have a clearer name? `RecurringContribBodyCopy`

**Pull request merged**